### ISSUE TITLE
Accessibility: Some links don't have discernible text

### ISSUE THREAD

**Issue by bedmison** [web-cat/code-workout#240]
Links must have discernible text. This shows up in the edit icon in an exercise. 

---

From the WCAG 2.1 spec:

To solve this problem, you need to fix at least (1) of the following:

- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute

And fix the following:
Element is in tab order and does not have accessible text
